From foo@baz Wed Jan 21 08:15:58 PST 2009
Date: Wed, 21 Jan 2009 08:15:58 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: sxg: fix build warnings in downloadB firmware files

From: Greg Kroah-Hartman <gregkh@suse.de>

These variables are never used, so #ifdef them away.

This should probably be fixed up properly from someone who knows why we
are even including these files in the first place, when they do not seem
to be needed at all.

Cc: Mithlesh Thukral <mithlesh@linsyssoft.com>
Cc: LinSysSoft Sahara Team <saharaproj@linsyssoft.com>
Cc: Christopher Harrer <charrer@alacritech.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/sxg/saharadbgdownloadB.c |    2 ++
 drivers/staging/sxg/saharadownloadB.c    |    2 ++
 2 files changed, 4 insertions(+)

--- a/drivers/staging/sxg/saharadbgdownloadB.c
+++ b/drivers/staging/sxg/saharadbgdownloadB.c
@@ -2,6 +2,7 @@
 #define SAHARA_B_UCODE_VERS_DATE  			"$Date: 2008/08/19 00:05:59 $"
 #define SAHARA_B_UCODE_HOSTIF_ID  			3
 
+#if 0
 static u32 	SBNumSections = 0x1;
 static u32  	SBSectionSize[] =
 {
@@ -12,3 +13,4 @@ static u32  	SBSectionStart[] =
 	0x00000000, 0x00001fff, };
 
 static unsigned char SaharaUCodeB[1][1];
+#endif
--- a/drivers/staging/sxg/saharadownloadB.c
+++ b/drivers/staging/sxg/saharadownloadB.c
@@ -2,6 +2,7 @@
 #define SAHARA_B_UCODE_VERS_DATE  			"$Date: 2008/08/19 00:05:59 $"
 #define SAHARA_B_UCODE_HOSTIF_ID  			3
 
+#if 0
 static u32 	SBNumSections = 0x1;
 static u32  	SBSectionSize[] =
 {
@@ -12,3 +13,4 @@ static u32  	SBSectionStart[] =
 	0x00000000, 0x00001fff, };
 
 static unsigned char SaharaUCodeB[1][1];
+#endif
